mesheryctl/pkg/utils/error.go

```diff
@@ -195,6 +195,43 @@ func RelationshipsError(msg string, cmd string) string {
 	}
 }
 
+func ComponentSubError(msg string, cmd string) string {
```

Comment:
Where exactly this function is referenced?

Reply:
This is not needed for exp, I was confusing it with another command. Fixed, thanks.